README.md

```diff
@@ -0,0 +1,11 @@
+
+
+
+To recreate the PDF file, first install the Ubuntu following packages:
+
+```sudo apt-get install texlive-base texlive-latex-base texlive-latex-extra texlive-fonts-extra texlive-science```
+
+Then type `make`.  
+
+You may have to press [ENTER] a couple of times to skip past some error messages.
```

Comment:
This seems like it should never have been acceptable.
https://github.com/riscv/riscv-trace-spec/pull/23 should fix that once it's approved and merged.